<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SimpleQtOpenGLAppClass</class>
 <widget class="QMainWindow" name="SimpleQtOpenGLAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SimpleQtOpenGLApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="OpenGLWidget" name="openglWidget" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>测试</string>
    </property>
    <addaction name="actionDisplayTriangle"/>
    <addaction name="actionImportModel"/>
    <addaction name="actionClearScene"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionDisplayTriangle">
   <property name="text">
    <string>显示三角形</string>
   </property>
  </action>
  <action name="actionImportModel">
   <property name="text">
    <string>导入模型</string>
   </property>
  </action>
  <action name="actionClearScene">
   <property name="text">
    <string>清空场景</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>OpenGLWidget</class>
   <extends>QWidget</extends>
   <header>OpenGLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="SimpleQtOpenGLApp.qrc"/>
 </resources>
 <connections/>
</ui>
